pkg/api/server/register_images.go

```diff
@@ -762,6 +762,29 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
 	//   500:
 	//     $ref: '#/responses/InternalError'
 	r.Handle(VersionedPath("/libpod/images/{name:.*}/exists"), s.APIHandler(libpod.ImageExists)).Methods(http.MethodGet)
+	// swagger:operation GET /libpod/images/{name}/resolve libpod ImageResolveLibpod
```

Comment:
First choice,
```suggestion
	// swagger:operation GET /libpod/images/{name}/qualifier libpod ImageQualifierLibpod
```
Second choice,
```suggestion
	// swagger:operation GET /libpod/images/{name}/resolution libpod ImageResolutionLibpod
```

Nouns are more RESTful than verbs. I know we inherited a lot of endpoints where that is not true. And neither of these "sing" to me.